public/js/loaders/level.js

```diff
@@ -85,6 +85,7 @@ export function createLevelLoader(entityFactory) {
             const level = new Level();
             level.name = name;
             level.music.setPlayer(musicPlayer);
+            level.playerPosition = levelSpec.playerPosition;
```

Comment:
This creates a reference to the levelSpec from the level. We should copy the primitives instead as the playerPosition object would change if levelSpec was altered somewhere.